list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+$query = "SELECT * FROM books";
```

Comment:
Observations: SQL Injection vulnerability due to directly embedding user input into SQL queries.
Recommendations: Use prepared statements to safely handle user input.